README.md

```diff
@@ -65,6 +65,12 @@ widely used, while draft-03 is long since deprecated.
 If you see anything missing from the current supported drafts, or incorrect
 on any draft still accepting bug fixes, please file an issue or submit a PR.
 
+Optional Tests
+--------------
+
+Tests in the `optional/` directory test optional validation behavior. Validators
```

Comment:
Thanks! I was hoping to even say a bit about each of the files present here -- e.g. a list of each file we have, and an explanation/link to why it's optional (e.g., for `zeroTerminatedFloats.json`, it's for languages that do not have floating point types. For `format`, it's because it being a validation keyword rather than informative is optional, etc.

This way there's a short sentence about each and a list of why they're not required.

Does that make sense?

Reply:
Could this be put into the test data itself, either in the 'description' field or in an additional 'comment' field?

Reply:
Decent sounding idea to me.